RELEASE_NOTES.md

```diff
@@ -1,21 +1,18 @@
-# Cosmos SDK v0.46.13 Release Notes
+# Cosmos SDK v0.46.14 Release Notes
 
-This release includes few improvements and bug fixes.
-Notably, the [barberry security fix](https://forum.cosmos.network/t/cosmos-sdk-security-advisory-barberry/10825). All chains using Cosmos SDK v0.46.0 and above must upgrade to `v0.46.13` **immediately**. A chain is safe as soon as **33%+1** of the voting power has upgraded. Coordinate with your validators to upgrade as soon as possible.
-
-Additionally, it includes new commands for snapshots management and bootstrapping from a local snapshot (add `snapshot.Cmd(appCreator)` to the chain root command for using it).
+<!-- todo -->
```

Comment:
this will be in a follow up pr or?

Reply:
Yeah, this is filled in the PR that prepares the release notes.